pkg/monitoring/deploy.go

```diff
@@ -120,6 +120,12 @@ func DeployApp(cattleAppClient projectv3.AppInterface, projectID string, createO
 	} else {
 		app = app.DeepCopy()
 		app.Spec.Answers = createOrUpdateApp.Spec.Answers
+
+		// clean up status
+		if forceRedeploy {
+			app.Status = projectv3.AppStatus{}
```

Comment:
Instead of cleaning the app status, I would prefer adding a timestamp parameter to answers.